backend/users/admin.py

```diff
@@ -1,10 +1,31 @@
+import csv
+from io import TextIOWrapper
+
 from django.contrib import admin
 from django.contrib.auth.admin import UserAdmin
+from django.urls import path
+from django.shortcuts import render, redirect
 
-from .forms import CustomUserCreationForm, CustomUserChangeForm
+from .forms import CustomUserCreationForm, CustomUserChangeForm, BulkImportForm
 from .models import CustomUser, Faculty, Student, Department
 
 
+def add_student(row):
+    email = row['email']
+    id_num = row['id number']
+    name = row['name']
+    Student.objects.create(email=email, id_num=id_num, name=name)
```

Comment:
Unnecessary variable creation. Define dicts with expected csv headers and use `**` unpacking to directly pass the args to the `create` function.